azure/v12.1.0/release.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v12.1.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
```

Comment:
In April, we drafted release notes for this and cert-exporter was v1.2.2 to make Helm 3 compatible. We didn't do this anymore? https://www.dropbox.com/scl/fi/rcle9tj1oj54netb5sqkk/Helm-3-Release-Notes.paper?dl=0&rlkey=nq013qe7hn8r0lh7iasys1mm1

Reply:
@cokiengchiara Nice catch! This is weird though as I'd expect the app to fail to install without this change.

@tomahawk28 Can you take a look?

Reply:
good catch, I think helm 2to3 doesn't report any errors from initial migration. I will change the spec of 12.1.0 as we already surveyed in the doc. thanks!

Reply:
I will upgrade cert-exporter to 1.2.4 since that's the latest version.

Reply:
looks like cert-exporter is upgraded to 1.2.4. resolving.